src/seedu/addressbook/data/person/Address.java

```diff
@@ -23,7 +20,7 @@
     public Address(String address, boolean isPrivate) throws IllegalValueException {
         String trimmedAddress = address.trim();
```

Comment:
Since the `Contact` superclass already does the validation of the values, could the subclass (in this case `Address`) also use a `super` constructor (i.e. `super(address, isPrivate)` to achieve the same thing while keeping in mind DRY principle?